client/cmd/dexc/sample-dexc.conf

```diff
@@ -0,0 +1,99 @@
+[Application Options]
+
+; ------------------------------------------------------------------------------
+; Data settings
+; ------------------------------------------------------------------------------
+
+; Path to application directory. The default is ~/.dexc on UNIX OSes and
+; ~/Library/Application Support/Dexc on macOS.
+; appdata=~/.dexc
+
+; Path to an INI configuration file. The default is ~/.dexc/dexc.conf.    
+; config=~/.dexc/dexc.conf
+
+; Database filepath. Database will be created if it does not exist. The default
+; dir is network specific:
+; Mainnet:
+; db=~/.dexc/mainnet/dexc.db
+; Testnet:
+; db=~/.dexc/testnet/dexc.db
+; Simnet:
+; db=~/.dexc/simnet/dexc.db
+
+; ------------------------------------------------------------------------------
+; Network settings
+; ------------------------------------------------------------------------------
+
+; Use testnet.
+; testnet=true
+
+; Use simnet.
+; simnet=true
+
+; Connect via a SOCKS5 proxy.
+; torproxy=127.0.0.1:9051
+
+; Enable Tor stream isolation by randomizing proxy user credentials resulting in
+; Tor creating a new circuit for each connection.  This makes it more difficult
+; to correlate connections.
+; torisolation=true
+
+; ------------------------------------------------------------------------------
+; RPC server settings
+; ------------------------------------------------------------------------------
+
+; Turn on RPC server.
+; rpc=true
+
+; RPC server listen address. The default value is network specific:
+; Mainnet:
+; rpcaddr=127.0.0.1:5757
+; Testnet: 
+; rpcaddr=127.0.0.2:5757
+; Simnnet:
+; rpcaddr=127.0.0.3:5757
+
+; RPC server user name.
+; rpcuser=user
+
+; RPC server password.
+; rpcpass=password
+
+; RPC server certificate file location.
+; rpccert=~/.dexc/rpc.cert
+
+; RPC server key file location.
+; rpckey=~/.dexc/rpc.key
+
+; ------------------------------------------------------------------------------
+; Web server settings
+; ------------------------------------------------------------------------------
+
+; HTTP server listen address. The default value is network specific:
+; Mainnet:
+; webaddr=127.0.0.1:5758
+; Testnet: 
+; webaddr=127.0.0.2:5758
+; Simnnet:
+; webaddr=127.0.0.3:5758
+
+; Disable the web server. This can be true only if RPC server
+; is on(rpc=true).
+; noweb=true
+
+; Reload the webserver's page template with every request. For development 
+; purposes.
+; reload-html=true
+
+; ------------------------------------------------------------------------------
+; Debug settings
+; ------------------------------------------------------------------------------
+
+; Logging level {trace, debug, info, warn, error, critical}
+; log=debug
+
+; Use local time zone time stamps in log entries.
+; loglocal=true
+
+; Display version information and exit
+; version=true
```

Comment:
We should omit `version` from the sample config because it only makes sense on the command like, just like the `--help` switch.

Reply:
yep thought the same forgot to ditch them